tests/cookiecutter.go

```diff
@@ -45,6 +46,10 @@ func (c *Cookiecutter) fillDefaultValue() {
 		c.UseHeroku = No
 	}
 
+	if c.UseOpenAPI != Yes && c.UseOpenAPI != No {
```

Comment:
I recommend not to have this option at all to keep the configuration simpler and  make Open API default if the project has APi variant to enforce adoption